### DISCUSSION TITLE
webhook with old style [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2330] [Discord Threads]
> 🙋‍♂️ **ofir**  
## why have on me old style design?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/981829050001657866/1a99889f-2a4e-4b17-ab82-b74fea38be15.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### please check your version  
#  
> 🙋‍♂️ **ofir**  
#### Node: **v16.15.0**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **sqlite3**   
#### PackageVersion: **0.91.7**  
#  
> 🙋‍♂️ **wkw**  
#### how did u start nocodb?  
#  
> 🙋‍♂️ **ofir**  
#### yes  
#  
> 🙋‍♂️ **wkw**  
#### i mean how  
#  
> 🙋‍♂️ **wkw**  
#### starting frontend & backend both locally?  
#  
> 🙋‍♂️ **ofir**  
#### how?  
#  
> 🙋‍♂️ **ofir**  
#### i restart app with pm2  
#  
> 🙋‍♂️ **wkw**  
#### can u try on another browser to see if it s related to cache  
#  
> 💁 **o1lab**  
#### just do a hard refresh - it will be fine  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981829050773438534)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
